infra/aws/terraform/prow-build-cluster/terraform.prod.tfvars

```diff
@@ -19,29 +19,19 @@ eks_admins = [
   "xmudrii"
 ]
 
-assume_role                 = true
-deploy_kubernetes_resources = true
+prow_build_cluster = true
 
-cluster_name    = "prow-build-cluster"
-cluster_region  = "us-east-2"
-cluster_version = "1.25"
-
-vpc_cidr                  = "10.0.0.0/16"
-vpc_secondary_cidr_blocks = ["10.1.0.0/16", "10.2.0.0/16"]
-vpc_public_subnet         = ["10.0.0.0/18", "10.0.64.0/18", "10.0.128.0/18"]
-vpc_private_subnet        = ["10.1.0.0/18", "10.1.64.0/18", "10.1.128.0/18"]
-vpc_intra_subnet          = ["10.2.0.0/18", "10.2.64.0/18", "10.2.128.0/18"]
+cluster_name               = "prow-build-cluster"
+cluster_region             = "us-east-2"
+cluster_version            = "1.25"
+cluster_autoscaler_version = "v1.25.0"
```

Comment:
I assume we want to keep those in dedicated files to test version upgrades without interfering with the other cluster. Why region is not in the shared file?

Reply:
We could also move those versions to shared file and add them only to `canary` while testing upgrades. With the current order of `-var-files` that you introduced in Makefile, env specific tfvars have a higher priority than shared one.

Reply:
Yes. We can keep both region and cluster name in the shared file.